sorts/timsort.py

```diff
@@ -73,10 +73,10 @@ def timsort(lst):
 
 
 def main():
-
-    lst = [5,9,10,3,-4,5,178,92,46,-18,0,7]
+    lst = [5, 9, 10, 3, -4, 5, 178, 92, 46, -18, 0, 7]
     sorted_lst = timsort(lst)
     print(sorted_lst)
```

Comment:
Perhaps adding some tests (assert statements at the very least) for a few different cases would help.

- [ ] `[]` -> `[]`
- [ ] `[1]` -> `[1]`
- [ ] `[2, 1]` -> `[1, 2]`
- [ ] `[1, 2]` -> `[1, 2]`
- [ ] `[3, 2, 1]` -> `[1, 2, 3]`
- [ ] `[1, 2, 5, 8, 3, 6, 9, 4, 7]` -> `[1, 2, 3, 4, 5, 6, 7, 8, 9]`